template_config.yml

```diff
@@ -18,6 +18,7 @@ check_stray_pulpcore_imports: true
 cherry_pick_automation: false
 ci_env: {}
 ci_trigger: '{pull_request: {branches: [''*'']}}'
+ci_update_branches: ["0.15", , "0.14", "0.10", "0.7"]
```

Comment:
I did some rough estimation here. What do we really need?

@pulp/ansible @pulp/galaxy